specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/batchInference.json

```diff
@@ -0,0 +1,1712 @@
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/EndpointDeploymentIdParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/EndpointNameParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "machineLearningServices.json#/definitions/MachineLearningServiceError"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/BatchDeploymentResource"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "BatchDeployment"
+        ],
+        "summary": "Delete Batch Inference deployment.",
+        "operationId": "BatchDeployments_Delete",
+        "x-ms-examples": {
+          "Delete BatchDeployment": {
+            "$ref": "./examples/deleteBatchDeployment.json"
+          }
+        },
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/EndpointDeploymentIdParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/EndpointNameParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "machineLearningServices.json#/definitions/MachineLearningServiceError"
+            }
+          },
+          "200": {
+            "description": "Success"
+          },
+          "204": {
+            "description": "Success"
+          }
+        }
+      }
+    },
+    "/mferp/managementfrontend/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/batchEndpoints/{endpointName}/deployments/{deploymentId}/jobs/{jobId}": {
```

Comment:
these /mferp/ URLs are not valid ARM URLs

Reply:
Can you please specify what's invalid here? 

This is the path and not the full URL, all the checks passed, and in this case, I have added 'jobs/{jobId}' to the deployment resource.

Reply:
These URIs are the URIs that the SDKs will use to call ARM.  This is not a valid ARM API.  It should start with "/subscriptions/...".  If you can call https://management.azure.com/mferp/managementfrontend/subscriptions/... and get a successful response please paste a network trace of that here

Reply:
Got it, I was looking at the last part of the path, instead of the first part, fixed it.